### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kkiirill**
[DEMO LINK](https://kkiirill.github.io/react_movies-list-fetch-movies/)

### CODE REVIEW

**Review by SeKaM0 — APPROVED** [review]
Great job

File: src/App.tsx [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-  state: State = {
-    movies: [],
-  };
+  const addMovie = useCallback((newMovie: Movie) => {
+    if (!movies.some(film => film.imdbID === newMovie.imdbID)) {
+      setMovies([...movies, newMovie]);
+    } else {
+      // eslint-disable-next-line no-alert
+      alert('This movie was selected');
+    }
```
Comment (HelenPetrovska) [inline src/App.tsx:15]: alert))

File: src/components/FindMovie/FindMovie.tsx [hunk 2]
```diff
@@ -1,54 +1,106 @@
...
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className="input is-danger"
+              value={title}
+              onChange={((e) => {
+                setTitle(e.target.value);
+              })}
```
Comment (HelenPetrovska) [inline src/components/FindMovie/FindMovie.tsx:64]: you can create method

**Review by HelenPetrovska — APPROVED** [review]
Great!

File: src/App.tsx (hunk 1 above)
Comment (danylo-onishchuk) [reply src/App.tsx:15]: no alert

File: src/api.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const getMovies = async (title: string) => {
+  const response = await fetch(`https://www.omdbapi.com/?apikey=9e088334&t=${title}`);
```
Comment (danylo-onishchuk) [inline src/api.ts:2]: creare variable for url and api key

**Review by danylo-onishchuk — CHANGES_REQUESTED** [review]

**Review by danylo-onishchuk — APPROVED** [review]

**Pull request closed without merging**